cli/src/test_utils.rs

```diff
@@ -46,15 +44,15 @@ pub fn wait_n_slots(rpc_client: &RpcClient, n: u64) -> u64 {
     loop {
         sleep(Duration::from_millis(DEFAULT_MS_PER_SLOT));
         let new_slot = rpc_client.get_slot().unwrap();
-        if new_slot - slot > n {
+        if new_slot.saturating_sub(slot) > n {
```

Comment:
Totally aside, but I never realized this is technically `wait_n_plus_one_slots`

Reply:
I think it is a bug.
It is only used in the CLI tests and from what I can understand from the tests, they do not expect it to be `n + 1`.

One set of tests wait one slot to work around the program deployment limitation.  1 slot is enough there.

The second set of tests is for staking.
I do not understand staking all that well, but here is a usage example:
```rust
    // to `stake2_keypair`
    check_balance!(rent_exempt_reserve, &rpc_client, &stake_keypair.pubkey());
    check_balance!(50_000_000_000, &rpc_client, &stake2_keypair.pubkey());

    // wait for new epoch plus reward blocks
    wait_for_next_epoch_plus_n_slots(&rpc_client, 1);
```

Let me send a PR to `s/>/>=/`.

Reply:
Thanks! The staking example should behave correctly when waiting just one slot, since rewards payout is meant to take one block with partitioned rewards